src/main/java/executor/service/config/beans/ThreadPoolConfigFactory.java

```diff
@@ -0,0 +1,35 @@
+package executor.service.config.beans;
+
+import executor.service.config.properties.PropertiesConfig;
+import executor.service.model.ThreadPoolConfig;
+
+import static executor.service.config.properties.PropertiesConstants.*;
+
+//ToDO: not sure whether it should exist separately from ThreadPoolFactory
+public class ThreadPoolConfigFactory {
+
+    private static ThreadPoolConfig poolConfig;
+
+    public static ThreadPoolConfig getPoolConfig() {
```

Comment:
1. DI will create a singleton of this class and inject it. Why such an implementation? Where does the concurrency come from?
2. I would draw your attention to the fact that this class will be tested using PowerMockito, which is not very simple.